docs/source/commands/persisted-queries.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: The Rover persisted-queries command
+description: Publish persisted query manifests to your persisted queries list
+minVersion: 0.17.2
+---
+
+import AuthNotice from '../../shared/auth-notice.mdx';
+
+With [GraphOS Enterprise](/graphos/enterprise/), you can enhance a graph's security by maintaining a **persisted query list** (**PQL**) for your graph's self-hosted router. The [Apollo Router](/router/) checks incoming requests against the PQL, a safelist of trusted operations made by your first-party apps.
```

Comment:
kind of a small nit here and I don't love my wording change but I want to communicate here that you can use PQLs without enabling full safelisting in the router. what do you think?

```suggestion
With [GraphOS Enterprise](/graphos/enterprise/), you can enhance a graph's security by maintaining a **persisted query list** (**PQL**) for your graph's self-hosted router. The [Apollo Router](/router/) checks incoming requests against the PQL, and can be configured to reject operations that are not in the list.
```